fr/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+  <title>Ne demande pas la permission de demander, demande juste.</title>
```

Comment:
I am not so sure about this translation as well. the `just` in `just ask` is a synonym to `simply ask` not to `ask correctly`

Reply:
The sentence cannot be perfectly translated, but I think this the best translation. I will ask my friends if they have another idea.